completions.mjs

```diff
@@ -26,21 +26,16 @@ const openai = new OpenAI({
 });
 
 export async function fetchTextCompletion(tone, prompt, language) {
-    const completion = await openai.chat.completions.create({
-        model: 'gpt-4o-mini',
+    let completion = await openai.chat.completions.create({
+        model: "gpt-4o-audio-preview",
```

Comment:
It's generally a good idea to externalize this variable to make it easy to change this without requiring a code change. The same way you manage API keys and the PORT number

Reply:
Understood, thank you for the tip! Updated.